code/modules/clothing/under/jobs/security.dm

```diff
@@ -49,6 +49,15 @@
 	icon_state = "warden_darkblue"
 	worn_state = "warden_darkblue"
 
+/obj/item/clothing/under/rank/warden/remote
+	name = "remote warden's uniform"
+	desc = "It's made of a slightly sturdier material than standard jumpsuits, to allow for more robust protection. It has the words \"Remote Warden\" written on the shoulders."
```

Comment:
i'd add something about how it's attached to the chassis or something like that here